Pipfile

```diff
@@ -6,6 +6,11 @@ verify_ssl = true
 [packages]
 dataclasses = {version = ">=0.6",python_version = "< 3.7"}
 attrs = "*"
+adaptivealerting = {git = "https://github.com/ExpediaDotCom/adaptive-alerting-detector-build.git"}
```

Comment:
adaptivealerting should be an editable dev dependency.  https://pipenv-fork.readthedocs.io/en/latest/basics.html#editable-dependencies-e-g-e.  You can update the Pipfile with ```pipenv install --dev -e .```

Reply:
Thanks @brett-miller -- I'll take a look